pkgs/build-support/dlang/builddubpackage/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
```

Comment:
If this builder is still quite limited, please add a comment explaining its limitations so that others don't get confused when it doesn't work.